config/webpack/react/webpack.config.base.js

```diff
@@ -57,47 +57,47 @@ module.exports = {
             loader: 'file-loader',
             options: {
               limit: 8192,
-              name: 'fonts/[name].[hash:8].[ext]'
-            }
-          }
-        ]
-      }
-    ]
+              name: 'fonts/[name].[hash:8].[ext]',
+            },
+          },
+        ],
+      },
+    ],
   },
   optimization: {
     runtimeChunk: {
-      name: (entrypoint) => `runtime.${entrypoint.name}`
+      name: entrypoint => `runtime.${entrypoint.name}`,
```

Comment:
Maybe `manifest/runtime` should be enough here, why should we care about that?

Reply:
Zhuangbility.